src/background/onboarding.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import { store } from 'hybrids';
+import Options from '/store/options.js';
+
+(async () => {
+  const options = await store.resolve(store.get(Options));
+  const version = chrome.runtime.getManifest().version;
+
+  if (!options.version) {
```

Comment:
is this condition sufficient? shouldn't we retrigger the onboarding if the uses did not interact with it ?

Reply:
As we decided, the onboarding is shown only once after a clean installation (or updated from the previous version - which has not yet set that value). I am closing the conversation.